templates/zerver/help/send-a-group-of-people-a-private-message.md

```diff
@@ -1,75 +1,64 @@
 # Private Group Messages
-## There are three ways to send a private message to a group of people:
-### First Way:
-To send a new private group message,
-first click on the `New private message`
-button located at the bottom of your screen.
+You can send a private message by using the compose box, the group PM list,
+or private messages tab.
+### Sending a private message through the Compose box
 
-![New private message](/static/images/help/private-message.png)
+1. Click on the **New private message** button located at the bottom of your
+screen. This will make the **compose box** appear at the bottom center of your
+screen.
 
-The following box will be displayed:
+    ![New private message](/static/images/help/private-message.png)
 
-![New stream message](/static/images/help/private-box.png)
-
-In the **one or more people...** field,
+2. In the **one or more people...** field of the **compose box**,
 input the usernames or email addresses of the users
 you want to send a private group message to.
-Each username or email address should be separated by a comma (`,`).
+Each username or email address should be separated by a comma (**,**).
 When you begin typing an email address or username,
 autocomplete suggestions will appear for individuals.
 The list of users in this field creates the group
 to which the private message will be sent.
 
-If you attempt to input a nonexistent username or email address,
-an error message will appear.
+    ![New stream message](/static/images/help/private-box.png)
+
+3. Type your message to that group of users in the
+**Compose your message here...** field. You can now send your private message.
+
+!!! warn ""
+    If you attempt to input a nonexistent username or email address,
+    an error message will appear.
 
-### Second Way:
-If you have already private messaged a group of people,
+### Sending a private message through the Group PMs list
+1. If you have already private messaged a group of people,
```

Comment:
This doesn't feel like a step. This just gives information about the second method of sending private messages. I think the first step should be "Click on one of the groups".